pkg/config/config.go

```diff
@@ -95,3 +95,21 @@ func DefaultConfig() Config {
 		},
 	}
 }
+
+// FromPath creates a Config based on a path to a .toml file.
+func FromPath(path string) (Config, error) {
+	tree, err := toml.LoadFile(path)
+	if err != nil {
+		return Config{}, err
+	}
+	return decode(tree, nil)
+}
+
+// FromString creates a Config using a string representing the configuration formatted as a TOML document.
+func FromString(tomlData string) (Config, error) {
```

Comment:
Maybe have a `FromReader(io.Reader) (Config, error)` method that actually calls the toml lib, and have the other methods call that...

Reply:
I've introduced a Store and added a `fromStore` function which are called by the other functions. As I think the data source here can't be limited to the config file (we also have env vars).